### DISCUSSION TITLE
Use YOLO in commercial projects

### DISCUSSION THREAD

**Discussion by Everlasting-AM** [ultralytics/ultralytics#14203] [Q&A]
Hi!

I have been working for a few months on training multiple YOLO models for different purposes, and I wanted to make sure if it is possible to use these models to create and deploy services, even if their use can later be sold, for example, by providing access to a vehicle impact detection system in exchange for a monthly fee.

Thanks!

**Comment by pderrenger** [comment]
@Everlasting-AM hello!

Thank you for your interest in using YOLO models for your commercial projects. To use YOLO models in a commercial setting, such as deploying a vehicle impact detection system with a subscription fee, you will need to obtain an Ultralytics Enterprise License. This license allows you to integrate Ultralytics software and AI models into commercial products without the open-source requirements of the AGPL-3.0 license.

For more details and to apply for an Enterprise License, please visit [Ultralytics Licensing](https://ultralytics.com/license).

If you have any further questions or need assistance with the licensing process, feel free to reach out. We're here to help!
